scripts/metadata_comparison/test/lib/test_argument_regex.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import unittest
```

Comment:
unittest is a nice built-in. But nowadays most people use pytest. It might be worth to take a look. It makes testing quite easy.

Reply:
Might be too big of a refactor for this PR, but that's good to know about, especially if testing gets more complex and `unittest` isn't able to do what we need